### DISCUSSION TITLE
Ghelper always turn to Standard GPU Mode at start up everytime, an option to choose Eco mode at start up

### DISCUSSION THREAD

**Discussion by KelvinBrowski** [seerge/g-helper#1379] [Ideas]
i wanted Ghelper to start up with Eco mode instead of Standard Mode

**Comment by seerge** [comment]
@KelvinBrowski if you have 2023 model, it's done intentionally to [prevent some issues afterwards ](https://github.com/seerge/g-helper/discussions/1042)

You can turn this off under Extra settings :)
